2018 summer events

dasanchez opened the pull request:
Added the following dates:
May 17: Planning session @ HackLab.TO
June 21: Install party @ Free Geek
July 26: Install party @ HackLab.TO
August 23: Install party @ Free Geek

dasanchez committed:
Planning session and install parties

May 17: Planning session @ HackLab.TO
June 21: Install party @ Free Geek
July 26: Install party @ HackLab.TO
August 23: Install party @ Free Geek

dasanchez committed:
Added blurb on e-waste

Free Geek install parties have a call for e-waste donations

dasanchez committed:
E-waste blurb wording

dcwalk commented on _posts/2018-06-21-install-party.md:
```diff
@@ -0,0 +1,17 @@
+
+Do you have any e-waste you would like to donate to Free Geek? Bring it! See their list of [accepted electronics here](https://www.freegeektoronto.org/pick-up/).
+
+Planning on coming? Let us know through our [chat channel](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or email us at <hello@tomesh.net>.
```
Use the {{ site.email }} variable?

dcwalk commented on _posts/2018-07-26-install-party.md:
```diff
@@ -0,0 +1,15 @@
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party. Come get your mesh node set up and play with distributed applications! This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session.
+
+Planning on coming? Let us know through our [chat channel](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or email us at <hello@tomesh.net>.
```
Use the {{ site.email }} variable?

dcwalk commented on _posts/2018-08-23-install-party.md:
```diff
@@ -0,0 +1,17 @@
+
+Do you have any e-waste you would like to donate to Free Geek? Bring it! See their list of [accepted electronics here](https://www.freegeektoronto.org/pick-up/).
+
+Planning on coming? Let us know through our [chat channel](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or email us at <hello@tomesh.net>.
```
Use the {{ site.email }} variable?

dcwalk reviewed (commented):
Just a thought -- July and August tend to be pretty quiet months.

I'm just wondering whether July (which will have potentially 1. Maker Fest, 2. Our Networks, 3. People at HOPE over the first three weekends) might be okay to skip, or reconfigure into like just a social event?

Meet and greet for new to mesh people that can be mentioned at all those events?

darkdrgn2k commented:
Maker fest deadline is this Monday and as far as I know no one has submitted yet so I don't know if we will be there... 

Just a though but maybe doing a install night as part of state of our networks...?

dcwalk commented:
@darkdrgn2k -- I don't think an install _night_ will fit within the current schedule for our networks Fri-Sun. I think during the sprints (Mon-Wed) there was early discussion of tomesh hacking, which would basically do the same thing

dasanchez committed:
Email link

Install party posts use the {{site.email}} variable now

dasanchez commented:
@dcwalk I'm OK with making the July meetup at HackLab a Meet and Greet. Or would you prefer skipping HackLab altogether to meet at the Rhino or elsewhere instead?

dcwalk commented:
@dasanchez -- I think however you want to proceed is fine! (Sorry for relentless proposing!)

garrying commented on _posts/2018-05-17-planning-session.md:
```diff
@@ -0,0 +1,21 @@
+
+We will break off to discuss ongoing deployment efforts afterwards.
+
+We will meet at {{page.startTime | date:"%-I:%M %P "}} in [{{ page.location }}]({{ page.locationLink }}).  
```
2 trailing spaces at the line of this line.

garrying commented on _posts/2018-06-21-install-party.md:
```diff
@@ -0,0 +1,17 @@
+title: "Install Party"
+slug: install-party
+text: 'Bring your hardware to set up a mesh node'
+location: Free Geek Toronto, 180 Sudbury St  
```
2 trailing spaces at the line of this line.

garrying commented on _posts/2018-08-23-install-party.md:
```diff
@@ -0,0 +1,17 @@
+title: "Install Party"
+slug: install-party
+text: 'Bring your hardware to set up a mesh node'
+location: Free Geek Toronto, 180 Sudbury St  
```
2 trailing spaces at the line of this line.

dasanchez committed:
July meetup changes, removed trailing spaces

- July 26 meetup at HackLab is now a Meet and Greet session
- Removed trailing spaces pointed out by @garrying

dasanchez commented:
Thanks @garrying ! I removed those trailing spaces.
@dcwalk never relent 😉

Shrinks99 committed:
Planning Session Overview

Added more specific text related to the activities and discussions that will be happening at this planning session.

Changed end text to direct new members towards our install nights.  Hopefully this will be a more applicable introduction.

Shrinks99 commented:
Changed the message for the planning session, this should be more applicable to what we are discussing this time around (and also what I hope to accomplish at this one ;P )
